xbmc/input/InputManager.cpp

```diff
@@ -0,0 +1,406 @@
+/*
+*      Copyright (C) 2005-2014 Team XBMC
+*      http://xbmc.org
```

Comment:
I chose to leave the copyright as-is since there's not really any real new code in the files.